src/configs/template_GAN_train_config.json

```diff
@@ -57,18 +55,22 @@
         }
     },
     "losses": {
-        "0": {
-            "type": "HingeLossG",
+    },
+    "gan_losses": {
+        "generator": {
+            "type": "GANLoss",
             "args": {
                 "nickname": "hinge_loss_generator",
```

Comment:
By default values of class `GANLoss` constructor, the nickname should be `LSGAN_loss_genrator`